client/a8c-for-agencies/sections/purchases/licenses/license-details/style.scss

```diff
@@ -120,6 +120,11 @@
 	font-weight: 500;
 	line-height: 1.375;
 
+	&:hover {
+		background-color: #fafafa;
```

Comment:
If it already exists... Could we use some of the existing color vars? from the color theme.